FurnitureShop/src/FurnitureShop.Admin.Api/Services/UserService.cs

```diff
@@ -28,10 +28,10 @@ public async Task DeleteUser(Guid userId)
 
         public async Task<UserView> GetUserById(Guid userId)
         {
-            var userview = unitOfWork.AppUsers.Find(user => user.Id == userId).Adapt<UserView>();
-            if (userview is null)
+            var user = unitOfWork.AppUsers.GetById(userId);
```

Comment:
You can use
`var user = _userManager.FindAsync(userId);`
or
`var user = _userManager.Users.FirstOrDefault(userId);`
instead of `var user = unitOfWork.AppUsers.GetById(userId);`